python/src/FunctionalChaosValidation_doc.i.in

```diff
@@ -0,0 +1,165 @@
+cross-validation methods presented in :ref:`pce_cross_validation`.
+
+Analytical cross-validation can only be performed accurately if some
+conditions are met.
+
+- This can be done only if the coefficients of the expansion are estimated
+  using least squares regression: if the expansion is computed from integration,
+  then an exception is produced.
+- This can be done only if the coefficients of the expansion are estimated
+  using full expansion, without model selection: if the expansion is computed
+  with model selection, then an exception is produced by default.
+  This is because model selection leads to supposedly improved coefficients,
+  so that the hypotheses required to estimate the mean squared error
+  using the cross-validation method are not satisfied anymore.
+  As a consequence, using the analytical formula without taking into
+  account for the model selection leads to a biased, optimistic, mean squared
+  error.
+  More precisely, the analytical formula produces a MSE which is lower
+  than the true one on average.
+  Model selection is involved in the expansion if the
+  :class:`~openturns.LeastSquaresMetaModelSelectionFactory`
+  or :class:`~openturns.CleaningStrategy` classes are involved.
+  If the :class:`~openturns.PenalizedLeastSquaresAlgorithmFactory` class
+  is used, then no model selection is involved and the analytical
+  cross-validation methods can be used.
+  If model selection is involved, the naive methods based on the
+  :class:`~openturns.LeaveOneOutSplitter` and :class:`~openturns.KFoldSplitter`
+  classes can be used, but this can be much slower than the
+  analytical methods implemented in the :class:`~openturns.experimental.FunctionalChaosValidation`
+  class.
+  In many cases, however, the order of magnitude of the estimate from the
+  analytical formula applied to a sparse model is correct: the estimate of
+  the MSE is only slightly lower than the true value.
+  In order to enable the calculation of the analytical MSE estimator
+  on a sparse model, please set the `FunctionalChaosValidation-ModelSelection`
+  key of the :class:`~openturns.ResourceMap` to `True`: use this
+  option at your own risks.
+
+We suggest to use leave-one-out (LOO) cross validation when possible,
+because it produces a more accurate estimate of the error than K-Fold does.
+If K-Fold is required, we suggest to use the largest possible value of :math:`k`.
+
+The predictions of the leave-one-one or K-Fold surrogate models
+are based on the hold-out method.
+For example, if we use the leave-one-out cross-validation method,
+the :math:`i`-th prediction is the prediction of the linear model
+trained using the hold-out sample where the :math:`i`-th observation
+was removed.
+This produces a sample of residuals which can be retrieved using
+the :class:`~openturns.experimental.FunctionalChaosValidation.getResidualSample` method.
+The :class:`~openturns.experimental.FunctionalChaosValidation.drawValidation` performs
```

Comment:
Is there a particular reason you are calling ":class:" instead of ":meth:" ?

Reply:
No, I think that this is an error here.

Reply:
Fixed.